rotkehlchen/chain/aggregator.py

```diff
@@ -957,7 +958,14 @@ def query_solana_balances(self, ignore_cache: bool) -> None:
                     usd_value=balance * native_token_usd_price,
                 )
 
-        # TODO: query token balances
+        for account in accounts:
+            token_balances = self.solana.get_token_balances(account)
+            token_prices = Inquirer.find_usd_prices(list(token_balances.keys()))
+            for token, balance in token_balances.items():
+                chain_balances[account].assets[token][DEFAULT_BALANCE_LABEL] = Balance(
+                    amount=balance,
+                    usd_value=balance * token_prices.get(token, ZERO_PRICE),
```

Comment:
```suggestion
                    usd_value=balance * token_prices[token],
```
zero price is set by default if no price is found.